travis_script.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-if [ "$TRAVIS_PULL_REQUEST" = "false" ]
+if [[ "$TRAVIS_PULL_REQUEST" = "false" ] && [ "$TRAVIS_BRANCH" = "qa" ]]
```

Comment:
This is a good idea , can you now take a look at the error logs /travis_script.sh: line 3, look through the script file